app/models/child.rb

```diff
@@ -0,0 +1,8 @@
+class Child < ApplicationRecord
+  belongs_to :need, counter_cache: :number_of_children
```

Comment:
Has anyone tested the behavior of re-using `number_of_children` here? For example, if an existing need was created with `number_of_children` set (before this is merged), are we at risk of it being cleared because there are in fact no associated Child records?